src/app.js

```diff
@@ -170,6 +231,38 @@ app.on('ready', () => {
     }
   }
 
+  function setNotificationTime(event, time) {
+    notificationWindow.close();
+    if (time && time !== store.get('notification.time')) {
+      let hours = time.split(':')[0];
+      let minutes = time.split(':')[1];
+      store.set('notification.time', time);
+      store.set('notification.hours', hours);
+      store.set('notification.minutes', minutes);
+      log.info(`Store updated - time=${time} hours=${hours} minutes=${minutes}`);
+      job.setTime(new CronTime(`0 ${minutes} ${hours} * * *`));
```

Comment:
The reason its not working is because job.setTime stops running the cronjob here. If we look at the source for node-cron we can see how setTime works. [setTime](https://github.com/kelektiv/node-cron/blob/25b8289b866aaebb14457ca5f2fa826bf679b59a/lib/cron.js#L535)

Reply:
Oh yeah, thank's ! When I wanted to check the definition with VSCode it redirected me on the `.d.ts` file and I was a bit lost because I don't know TS

Reply:
When I debug with `console.log(job.nextDates())` and `console.log(job.running)`it returns the correct time and `true` for `job.running` But the notification won't appear

Reply:
Okay, my bad I was in "Don't disturb" mode on my OS !

Reply:
I'll take a look at this tomorrow if you still haven't figured it out. It's late for me haha

Reply:
Oh haha ok good